public/javascripts/app.js

```diff
@@ -90,11 +85,14 @@ angular.module('uforia',
                 url: "/login",
                 templateUrl: "views/authentication/login.html",
                 controller: 'loginCtrl'
-                })
+            })
             .state('user', {
                 url: "/user",
                 templateUrl: "views/userOverview.html",
-                controller: 'userOverviewCtrl'
+                controller: 'userOverviewCtrl',
+                resolve: {
+                    role: isAdmin
```

Comment:
Wouldn't it be better to implement this like:
`roles: hasRoles(Constant.ADMIN, Constant.MANAGER)`.
How would add multiple roles to a page?